public/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css'>
+  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Carter+One'>
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="theme-color" content="#000000" />
+ 
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+  <title>Lodge.io</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/react/16.8.1/umd/react.production.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.8.1/umd/react-dom.production.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/axios/0.18.0/axios.min.js'></script>
+  <script src='https://unpkg.com/react-infinite-scroll-component@4.5.0/lib/index.js'></script>
```

Comment:
Most of these cdns can be installed as dependencies in npm and packaged with webpack.

Reply:
To maintain testing flexibility I wanted to keep these in for now and will remove when the integrations I'm working on are tighter.

Reply:
I think there are reasons to use a cdn as opposed to webpack, so as long as you're making the decision consciously its all good.